### PR TITLE
feat: new IBM QRadar alert channel resource

### PR DESCRIPTION
**Pull request by dmurray-lacework**
IBM QRadar alert channel resource

Signed-off-by: Darren Murray <darren.murray@lacework.net>

### CODE REVIEW

File: website/docs/r/alert_channel_qradar.html.markdown [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+layout: "lacework"
+page_title: "Lacework: lacework_alert_channel_qradar"
+description: |-
+  Create and manage IBM QRadar Alert Channel integrations
+---
+
+# lacework\_alert\_channel\_qradar
+
+You can configure an alert channel to receive Lacework alert notifications in IBM QRadar.
+
```
Comment (scottford-lw) [inline website/docs/r/alert_channel_qradar.html.markdown:12]:
```suggestion
You can configure an alert channel to send Lacework alert notifications to IBM QRadar.

```

**Review by scottford-lw — CHANGES_REQUESTED** [review]
@dmurray-lacework small suggestion

File: examples/resource_lacework_alert_channel_qradar/main.tf [hunk 2]
```diff
@@ -0,0 +1,8 @@
+provider "lacework" {}
+
+resource "lacework_alert_channel_qradar" "example" {
+  name      = "QRadar Channel Alert Example"
+  host_url = "https://qradar-lacework.com"
+  host_port = 4000
+  communication_type = "HTTPS"
```
Comment (afiune) [inline examples/resource_lacework_alert_channel_qradar/main.tf:7]:
```suggestion
  name               = "QRadar Channel Alert Example"
  host_url           = "https://qradar-lacework.com"
  host_port          = 4000
  communication_type = "HTTPS"
```

File: lacework/resource_lacework_alert_channel_qradar.go [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+			"org_level": {
+				Type:     schema.TypeBool,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func resourceLaceworkAlertChannelQRadarCreate(d *schema.ResourceData, meta interface{}) error {
+	comm, _ := api.QRadarComm(d.Get("communication_type").(string))
```
Comment (afiune) [inline lacework/resource_lacework_alert_channel_qradar.go:90]: Note to me, this is safe since we have a Validation function.

**Review by afiune — APPROVED** [review]
tenor-72077178

**Pull request merged**